### DISCUSSION TITLE
Github student ID verification is not happening

### DISCUSSION THREAD

**Discussion by payas12** [community/community#60095] [GitHub Education]
My name is payas from India , I had applied for students program of GitHub but regretted 3 times even after uploading valid college I'd card and proofs please help
#github
#58288
#59099

**Comment by anjylo** [comment]
I think you might want to contact Github support. 

https://support.github.com/
https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket

**Reply by payas12** [reply]
Thanks "Angelo" just before reading your email i got verified, thanks for the helping hand . 😁😸🙏🙏

**Reply by anjylo** [reply]
You're welcome. If you could also update the filter from "unanswered"  to "answered" or "close" is much appreciated. Easier to filter and find more unanswered questions. Also, you can mark/accept my/others answer. 👍

**Comment by payas12** [comment]
Thanks "Angelo" just before reading your email i got verified, thanks for
the helping hand . 😁😸🙏🙏

**Comment by samus-aran** [comment]
Hi @payas12 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!

**Reply by payas12** [reply]
Thanks for looking into this matter , and I have received the the benefits after 2 days. 

Thanks @samus-aran
